app/assets/javascripts/components/InvitationForm.coffee

```diff
@@ -59,24 +64,24 @@ invitationForm = React.createClass
           if @shouldShowErrorFor 'domainIds' then 'domain-wrapper invalid'
           else 'domain-wrapper'
         p
-          'Kies een domein om op te testen'
-        input
-          ul
-            className: 'domains'
-            domains.map (domain) ->
-              li
-                key: domain.id
-                className: 'domain'
-                input \
-                  merge domainIds,
-                        type: 'radio'
-                        name: 'domainIds'
-                        id: domain.id
-                        value: domain.id
-                label
-                  className: 'domain-label'
-                  htmlFor: domain.id
-                  domain.description
+          'Kies één of meerdere domeinen om op te testen'
```

Comment:
CATja is momenteel alleen in het Nederlands. Gewone strings maken wat duidelijker wat de code zal laten zien dan `t('models.domains.choose_one_or_more')` o.i.d.